tests/stockfish/test_models.py

```diff
@@ -914,6 +914,9 @@ def test_get_wdl_stats(self, stockfish: Stockfish):
             wdl_stats_3 = stockfish.get_wdl_stats()
             assert isinstance(wdl_stats_3, list) and len(wdl_stats_3) == 3
 
+            wdl_stats_4 = stockfish.get_wdl_stats(get_as_tuple=True)
+            assert isinstance(wdl_stats_4, tuple) and len(wdl_stats_4) == 3
+
```

Comment:
Here we could also test that `tuple(wdl_stats_3)` equals `wdl_stats_4`, and also `list(wdl_stats_4)` equals `wdl_stats_3`. From the changes you've made this should clearly be the case, but in the future the test will serve as a regression test in case a bad change is made to the `get_wdl_stats` function.